oktapam/provider.go

```diff
@@ -115,6 +117,7 @@ func Provider() *schema.Provider {
 		DataSourcesMap: map[string]*schema.Resource{
 			providerADConnectionsKey:                       dataSourceADConnections(),
 			providerCurrentUser:                            dataSourceCurrentUser(),
+			providerDatabaseKey:                            dataSourceDatabase(),
```

Comment:
generally we have a list-based datasource as well.  the usage is really more around obtaining the id for a resource based on a filter, so you may do something like: 

```
data oktapam_databases db-list {
  resource_group = '...'
  project = '...'
  name = 'YourMySQL'
}

data oktapam_database yourmysql {
  id = data.oktapam_databases.db-list[0].id
}
```

will we be supporting that?

Reply:
We will be supporting that. We need to add API for listing all database resources first.

Reply:
That makes sense, thanks @sachinsaxena-okta